pytest_selenium/pytest_selenium.py

```diff
@@ -198,6 +200,13 @@ def _gather_html(item, report, driver, summary, extra):
 
 
 def _gather_logs(item, report, driver, summary, extra):
+    pytest_html = item.config.pluginmanager.getplugin('html')
+    # Get geckodriver log if available
+    if item.config._driver_log is not None:
```

Comment:
I would expect this to raise an exception if the attribute is not set (it's only set for Firefox driver). We should probably use `hasattr` or `getattr` instead.